File: Java/dependency-injection/bastard-injection/src/main/java/com/github/grzesiek_galezowski/ConformingContainerAntipattern/_3_ConformingContainer/InMessages/NullMessage.cs
using ConformingContainerAntipattern._3_ConformingContainer.Interfaces;
using ConformingContainerAntipattern._3_ConformingContainer.Services;

namespace ConformingContainerAntipattern._3_ConformingContainer.InMessages
{
  class NullMessage : AcmeMessage
  {
    public void AuthorizeUsing(IAuthorization authorizationRules)
    {
      
    }

    public void WriteTo(DataDestination dataDestination)
    {
       
    }
  }
}

File: Java/dependency-injection/conforming-container-antipattern/src/main/java/com/github/grzesiek_galezowski/ConformingContainerAntipattern/InMessages/NullMessage.java
package com.github.grzesiek_galezowski.ConformingContainerAntipattern.InMessages;

import com.github.grzesiek_galezowski.ConformingContainerAntipattern.Interfaces.InboundMessage;
import com.github.grzesiek_galezowski.ConformingContainerAntipattern.Interfaces.DataDestination;
import com.github.grzesiek_galezowski.ConformingContainerAntipattern.Services.Authorization;

public class NullMessage implements InboundMessage {
  public void authorizeUsing(Authorization authorizationRules) {

  }

  public void writeTo(DataDestination dataDestination) {

  }
}
